packages/react-devtools-shared/src/__tests__/profilerContext-test.js

```diff
@@ -655,4 +655,223 @@ describe('ProfilerContext', () => {
 
     document.body.removeChild(profilerContainer);
   });
+
+  // @reactVersion >= 18
```

Comment:
No need to specify this. If you remove the pragma, it will only run the test against React from source.